low-level/files/opt/seagate/sspl/setup/resource_map/resource_map.py

```diff
@@ -16,7 +16,9 @@
 # please email opensource@seagate.com or cortx-questions@seagate.com
 
 import re
+import time
 from abc import ABCMeta, abstractmethod
+from framework.base.sspl_constants import DEFAULT_ALERT_RECOMMENDATION
```

Comment:
use DEFAULT_RECOMMENDATION, same as in #428

Reply:
Done.